docs/api/v4/changes.md

```diff
@@ -1,9 +1,32 @@
 # API v4 Changelog
 
-This page lists additions to the v4 api with each new release of CiviCRM Core.
+*This page lists additions to the v4 api with each new release of CiviCRM Core.*
 
 Also see: [Differences Between Api v3 and v4](/api/v4/differences-with-v3.md).
 
-## Nothing here yet.
+### 5.23 Added PaymentProcessor and PaymentProcessorType APIv4 Entities
 
-API v4 is brand new. No changes yet!
+See https://github.com/civicrm/civicrm-core/pull/15624
+
+### 5.23 $index param supports array input
+
+CiviCRM 5.23 supports two new modes for the `$index` param - associative and non-associative array. See https://github.com/civicrm/civicrm-core/pull/16257
```

Comment:
```suggestion
CiviCRM 5.23 supports two new modes for the `$index` param - associative and non-associative array. See [CiviCRM Core PR #16257](https://github.com/civicrm/civicrm-core/pull/16257) 
```

For accessibility it's nice to add context to links if we can with titles/text. The Material theme for the docs makes the links accessible if we add some text to them.